### PR TITLE
Create issue if snapshot build fails

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

File: .github/workflows/snapshot.yaml [hunk 1]
```diff
@@ -22,3 +22,24 @@ jobs:
 
       - name: Build and test
         run: ./gradlew -Psnapshot=yes build --scan --no-daemon
+
+  issue:
+    name: Open issue on failure
+    needs: [ build ]
+    runs-on: ubuntu-latest
```
Comment (mateuszrzeszutek) [inline .github/workflows/snapshot.yaml:29]:
```suggestion
    runs-on: ubuntu-20.04
```

File: .github/workflows/nightly-failed.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Nightly build #{{ env.GITHUB_RUN_NUMBER }} failed
+assignees: iNikem
```
Comment (mateuszrzeszutek) [inline .github/workflows/nightly-failed.md:3]: Should we put the maintainers group here instead?

File: .github/workflows/snapshot.yaml [hunk 3]
```diff
@@ -22,3 +22,24 @@ jobs:
 
       - name: Build and test
         run: ./gradlew -Psnapshot=yes build --scan --no-daemon
```
Comment (mateuszrzeszutek) [inline .github/workflows/snapshot.yaml:24]: > `-Psnapshot=yes`

Should we change how that flag works? Perhaps we should default to `snapshot=yes` and only pass `snapshot=no` when releasing a new stable version?

Comment (iNikem) [reply .github/workflows/snapshot.yaml:24]: I am not sure... Let's leave that as a separate though

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**